aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/UpdateHandler.java

```diff
@@ -128,7 +130,14 @@ protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
                     return versioned(proxy, rdsProxyClient, progress, null, ImmutableMap.of(
                             ApiVersion.V12, (pxy, pcl, prg, tgs) -> updateDbInstanceV12(pxy, request, pcl, prg),
                             ApiVersion.DEFAULT, (pxy, pcl, prg, tgs) -> updateDbInstance(pxy, request, pcl, prg)
-                    )).then(p -> checkFailedEvents(rdsClient, logger, p, p.getCallbackContext().getTimestamp(RESOURCE_UPDATED_AT)));
+                    )).then(p -> Events.checkFailedEvents(
+                            rdsClient,
```

Comment:
This parameter is called `rdsClient` here and `rdsProxyClient` in DBCluster. Let's unify these names.

Reply:
Done.